permission/permission.go

```diff
@@ -7,10 +7,13 @@ import (
 type Permission map[string]interface{}
 type Permissions map[string]Permission
 
-const Owner = "root"
-const Custodian = "custodian"
-const Write = "upload"
-const Read = "view"
+const (
+	Alerting  = "alerting"
```

Comment:
I think it's better to define this as `follow`, because if I am not mistaken, in addition to alerting we will allow the follower to read (a subset?) of the device the data of the followee.

Reply:
@ps2 do you have any objection? We had discussed "alerting" as the term earlier. Personally, I don't feel strongly either way.

Reply:
I think either works for me as well. We could also be specific and say "CarePartner".

Reply:
Will do. Going with Follow, as I feel it describes what it's allowing better than CarePartner.